multimedia/urls.py

```diff
@@ -1,7 +1,8 @@
 from django.conf.urls import patterns, include, url
 from django.contrib import admin
+from . import views
 
-urlpatterns = patterns('multimedia.views',
-    url(r'^books/$', 'book_list', name='book_list'),
-    url(r'^books/(?P<isbn13>\d{13})/', 'book_detail', name='book_detail'),
-)
+urlpatterns = [
+    url(r'^books/$', views.book_list, name='book_list'),
+    url(r'^books/(?P<isbn13>\d{13})/', views.book_detail, name='book_detail'),
+]
```

Comment:
It has always been this way since the beginning. But in Django 1.8 previous methods is deprecated.